src/test/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapperTest.java

```diff
@@ -287,6 +287,17 @@ public void upstreamBuilds() throws Throwable {
         });
     }
 
+    @Test
+    @Issue("JENKINS-73421")
+    public void externalizableId() throws Throwable {
+        sessions.then(j -> {
+            WorkflowJob first = j.createProject(WorkflowJob.class, "first-job");
+            first.setDefinition(new CpsFlowDefinition("echo currentBuild.getExternalizableId()\n", true));
```

Comment:
To be more robust, and also exercise the recommend JavaBeans-like property getter:
```suggestion
            first.setDefinition(new CpsFlowDefinition("echo(/externalizableId=$currentBuild.externalizableId/)", true));
```